### ISSUE TITLE
Support for netstandard2.1

### ISSUE THREAD

**Issue by sparkitny** [nblumhardt/influxdb-lineprotocol#95]
Hi there,

does anybody know whether this lib can be used with netstandard2.1?

**Comment by pocketbroadcast** [comment]
Yes, the assemblies in both nuget packages (InfluxDB.Collector and InfluxDB.LineProtocol) target .net standard 2.0

For .net version compatibility see: [https://docs.microsoft.com/en-us/dotnet/standard/net-standard](https://docs.microsoft.com/en-us/dotnet/standard/net-standard)
> The various .NET implementations target specific versions of .NET Standard. Each .NET implementation version advertises the highest .NET Standard version it supports, a statement that means it also supports previous versions.
